input.go

```diff
@@ -92,6 +92,29 @@ func IsMouseButtonPressed(mouseButton MouseButton) bool {
 	return uiDriver().Input().IsMouseButtonPressed(driver.MouseButton(mouseButton))
 }
 
+// GamepadGUID returns a string with the uuid.
+//
+// GamepadGUID is concurrent-safe.
+//
+// GamepadGUID always returns an empty string on browsers and mobiles.
+func GamepadGUID(id int) string {
+	return uiDriver().Input().GamepadGUID(id)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Gamepad/id should be available on browsers.

Reply:
ID is more analogous to glfw's GamepadName than GamepadGUID, so if you look at this PR, I'm actually already using ID.

`ebiten\internal\uidriver\js\input.go`
```
i.gamepads[id].name = gamepad.Get("id").String()
```

Reply:
Ah ok so there is no way to get GUID on browsers...